lua/auto-save/config.lua

```diff
@@ -27,6 +27,7 @@ Config = {
     condition = nil,
     write_all_buffers = false, -- write all buffers when the current one meets `condition`
     noautocmd = false, -- do not execute autocmds when saving
+    lockmarks = false, -- lock marks (mainly `[` and `]`) when saving
```

Comment:
Same thing as above.
```suggestion
    lockmarks = false, -- lock marks when saving, see `:h lockmarks` for more details
```